docs/tutorials/02-channel-upgrades/02-setup-env.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Set Up Your Work Environment
+sidebar_label: Set Up Your Work Environment
+sidebar_position: 2
+slug: /channel-upgrades/setup-env
+---
+
+import HighlightBox from '@site/src/components/HighlightBox';
+
+# Set up your work environment
+
+On this page, you can find helpful links to set up your work environment.
+
+<HighlightBox type="info" title="Dependencies">
+
+In this section, you can find all you need to install:
+
+- [gm](https://github.com/informalsystems/gm/)
+- [ibc-go simd](https://github.com/cosmos/ibc-go/)
+- [Hermes v1.9.0](https://hermes.informal.systems/)
```

Comment:
I guess `v1.9.0` will be out when this tutorial is out? Does this tutorial work at the moment?

Reply:
It works building hermes from the channel upgradeability branch. I added a warning to signal that the version of hermes with support is not released yet.